libraries/chain/database.cpp

```diff
@@ -88,7 +88,11 @@ database_impl::database_impl( database& self )
    : _self(self), _evaluator_registry(self) {}
 
 database::database()
-   : _my( new database_impl(*this) ) {}
+   : _my( new database_impl(*this) )
+{
+   ilog( "Checking STEEMIT_MIN_BLOCK_SIZE matches expected value of ${m}", ("m", signed_block::calculate_min_size()) );
```

Comment:
Does this really belong in the database constructor? The application startup logic seems like a more logical place.

Reply:
I don't want to introduce a merge conflict with `develop`.

Reply:
Please make an issue for 0.19.X so that we can fix this once the merge is complete.